src/components/BasicDetails.js

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 
-class PersonalInfo extends React.Component {
+class BasicDetails extends React.Component {
```

Comment:
Nice effort. :)

Even though you are using this for showing some basic details, you cant name it as `BasicDetails`, its raises some confusion.

Sugggestions
- `Input` 
- `TextField`
- `FormInput`

Have a look at **Material UI**. This is a react component library created by google https://material-ui.com/components/text-fields/ - They call it `FormField`

Another example is **Vue Storefront UI**, they call it `Input` https://docs.storefrontui.io/?path=/docs/components-atoms-input--common (this is in vuejs btw)